app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :user, class_name: 'User'
```

Comment:
- [ ] Good job so far, but the likes belong to `author` not `user`, kindly fix that 🙃

Reply:
Thanks for requesting changes, I am working on it 🙋🏿‍♂️.